BedrockServer.cpp

```diff
@@ -364,11 +364,49 @@ void BedrockServer::sync(const SData& args,
             server._suppressMultiWrite.store(true);
         }
 
-        // If the node's not in a ready state at this point, we'll probably need to read from the network, so start the
-        // main loop over. This can let us wait for logins from peers (for example).
+        // If the node's not in a ready state at this point, we'll probably need to read from the network. Test
+        // conditions from transitional states and then start the main loop over. This can let us wait for logins
```

Comment:
I'm open to improvements, but "so start the main loop over" doesn't make sense if we're about to drop into a bunch of additional code.

Reply:
Should `network. Test` be `network, test`, what do you mean by "transitional states"?

Reply:
Comma or not just looks like a writing style choice, IMO. I'll make that change if it resolves your concern. Alternatively, if i moved "Test" to its own line, my meaning might be a little clearer, as in "this is what is the code does, which is separate from the "why" we do this in the previous sentence.

By transitional states, I was attempting to generalize what "had a commit interrupted because of losing quorum" represents. Not very clear, I agree. Pushing a variation, LMK what you think.